.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Question about the project
+about: Issues are not limited to bugs or ideas. Ask your questions here too!
+title: ''
+labels: question
+assignees: ''
+
+---
+
+**The package and version I'm asking about**: 
+<!--
+There are a few of ways to find this out:
+* In a notebook, run the `#!about` magic command. 
+* At the command line, run `dotnet interactive --version`.
```

Comment:
```suggestion
* At the command line, run `dotnet interactive --version`. (_Non-VS Code only._)
```

Reply:
@jonsequitur someone who is going to ask a question will see a plain text, not a formatted one. So we probably don't need those `_`